src/modules/battery.rs

```diff
@@ -0,0 +1,77 @@
+use ansi_term::Color;
+
+use super::{Context, Module};
+
+/// Creates a segment for the battery percentage and charging state
+pub fn segment(_context: &Context) -> Option<Module> {
+    const BATTERY_FULL: &str = "•";
+    const BATTERY_CHARGING: &str = "⇡";
+    const BATTERY_DISCHARGING: &str = "⇣";
+    const BATTERY_THRESHOLD: f32 = 10.0;
+
+    let battery_status = get_battery_status()?;
+    let BatteryStatus { state, percentage } = battery_status;
+
+    if percentage > BATTERY_THRESHOLD {
```

Comment:
This is mostly for the sake of configuration. The user will provide a threshold for the percentage that the battery should dip below before the segment is shown. I thought it was more intuitive to provide `BATTERY_THRESHOLD = 10` than `BATTERY_THRESHOLD = 0.1`, but I'm open to feedback suggesting otherwise. 🙂

Reply:
Got it ... I had a misunderstanding as to what exactly the `BATTERY_THRESHOLD` was used for. Makes perfect sense the way that you implemented it!